### ISSUE TITLE
API returns redundant hiptest-uid column in feature file

### ISSUE THREAD

**Issue by IvanK-01** [SmartBear/cucumberstudio-issue-tracker#166]
### Steps to reproduce
1. In BDD-style project create some folder.
2. Add scenario with datatable.
3. Import feature file using https://studio-api.cucumber.io/#features API call.

### Expected behaviour
Call should return data without "hiptest-uid" column:
```
...
    Then I get "<result>" value from the test

    Examples:
      | result  |
      | empty   |
      | results |
```

### Actual behaviour
```
...
    Then I get "<result>" value from the test

    Examples:
      | result | hiptest-uid |
      | empty |  |
      | results |  |
```

**Comment by HoudaDabou** [comment]
Hi,

Thanks for this bug report :)

Our team take care of it and will correct it ASAP.

We'll keep you informed.

Regards,
Houda

**Comment by IvanK-01** [comment]
It is go away. Great, thank you! Now our projects are looked much better.

**Comment by vincent-psarga** [comment]
Great :)

**Comment by sabina-czopik-dragon** [comment]
It seems that this issue appears again. When I import feature file using https://hiptest.github.io/slate/#features API call or downloading a feature file from the Hiptest interface I get the following result 
`...
    Then I get "<result>" value from the test (<hiptest-uid>)

    Examples:
      | result | hiptest-uid |
      | empty |  |
      | results |  |`

**Comment by HoudaDabou** [comment]
Hi @sabina-czopik-dragon,

Can you send us the request that you sent to create your feature via the API and replacing your data by fake ones of course please?

**Comment by sabina-czopik-dragon** [comment]
Hi, I use following `hiptest-publisher -c hiptest-publisher.config`

**Comment by HoudaDabou** [comment]
Yes I had the same result, you're right...

So I'll ask the dev team what happened and get back to your ASAP.

**Comment by vincent-psarga** [comment]
> Hi, I use following `hiptest-publisher -c hiptest-publisher.config`
> hiptest-publisher.config
> 
> > token
> > not_recursive = false
> > split_scenarios = false
> > with_folders = true
> > push_format = cucumber-json
> > keep_filenames = false
> > keep_foldernames = false
> > empty_folders = false
> > leafless_export = false
> > no_uids = false
> > no_parent_folder_tags = false
> > with_dataset_names = false
> > language = cucumber
> > output_directory = './features'
> > framework = typescript
> > site = https://app.hiptest.com
> > step_definitions_output_directory = './features/step_definitions'
> > actionwords_output_directory = './features/step_definitions'

In you case, could you replace the line:

```
> > no_uids = false
```

by
```
> > no_uids = true
```

This should fix this issue normally.
Did you get the config file via the Hiptest automation page ?

**Comment by sabina-czopik-dragon** [comment]
it's helps, thx . I get the config file via the Hiptest automation page

**Comment by cbliard** [comment]
This issue was already fixed for the use cases described above, and has been fixed more generally in hiptest-publisher 2.3.0. Now when generating tests the hiptest-uid column is not present anymore, unless a test run id or name has been specified.
